### PR TITLE
fix: 🐛 gave autocomplete dropdown tooltips and larger size

### PR DESCRIPTION
**Pull request by gdoohen1**
Here is the ticket for these changes https://selectquote.atlassian.net/browse/SSC-134

The issue was the autocomplete's dropdown of options was limited to the field's size in the form which caused issues with readability of the options. I increased the width of the dropdown to be 2X the width of the field but with a limit so that it doesn't run off the screen. I also added tooltips to all the options.

**Loom**: https://www.loom.com/share/609c71187ca84dc6925ea88a5652e846
Note: The loom shows the tooltip text in middle of the row, I've updated it to be on the bottom-left so it now shows directly below the value for very large dropdowns.

### CODE REVIEW

**Comment by lucas-homer** [comment]
So if I understand correctly, this is now the default behavior. What if we had a flag or something to lock the width to the input width? I can imagine some situations where the dropdown options are not long, and they're positioned on the left of a form and we're going to end up with humongous dropdown windows. Thoughts?

**Comment by gdoohen1** [comment]
> So if I understand correctly, this is now the default behavior. What if we had a flag or something to lock the width to the input width? I can imagine some situations where the dropdown options are not long, and they're positioned on the left of a form and we're going to end up with humongous dropdown windows. Thoughts?

Yeah I think this is a good idea. Our stories I think are a bit atypical in that we have very large sizes for some of the fields, the fields in the actual CRM use `size={2}` for pretty much every field regardless of what it's for, so 2X width there is a bit less drastic but I agree this should definitely be able to be opted out of.

File: src/components/SQForm/SQFormAutocomplete.js [hunk 1]
```diff
@@ -109,9 +201,13 @@ function SQFormAutocomplete({
   onBlur,
   onChange,
   onInputChange,
-  size = 'auto'
+  size = 'auto',
+  lockWidthToField
```
Comment (SeanGroff) [inline src/components/SQForm/SQFormAutocomplete.js:205]: Can we default `lockWidthToField` to be false so we don't rely on `undefined`? Once this gets to TypeScript we'd have to make this adjustment anyway

**Review by SeanGroff — APPROVED** [review]
Great work @gdoohen1 💯

**Review by lucas-homer — APPROVED** [review]

**Review by codyarose — APPROVED** [review]

**Pull request merged**